flink-table/flink-table-planner-blink/src/main/java/org/apache/flink/table/planner/functions/sql/FlinkSqlOperatorTable.java

```diff
@@ -84,7 +85,7 @@ public static synchronized FlinkSqlOperatorTable instance() {
 	private static final SqlReturnTypeInference PROCTIME_TYPE_INFERENCE = createTimeIndicatorReturnType(false);
 
 	private static SqlReturnTypeInference createTimeIndicatorReturnType(boolean isRowTime) {
-		return ReturnTypes.explicit(factory -> {
```

Comment:
I think we should keep `ReturnTypes`, there are `ReturnTypes` and `InferTypes` and `OperandTypes`. It is confused if we remove class name.